modules/redis/src/main/scala/zio/redis/api/Streams.scala

```diff
@@ -64,32 +64,34 @@ trait Streams[G[+_]] extends RedisEnvironment[G] {
    *   ID of the stream
    * @param id
    *   ID of the message
+   * @param noMakeStream
+   *   The creation of stream's key can be disabled with the noMakeStream option
    * @param pair
    *   field and value pair
    * @param pairs
    *   rest of the field and value pairs
    * @return
    *   ID of the added entry.
    */
-  final def xAdd[SK: Schema, I: Schema, K: Schema, V: Schema](
-    key: SK,
-    id: I,
+  final def xAdd[SK: Schema, I: Schema, K: Schema, V: Schema](key: SK, id: I, noMakeStream: Boolean = false)(
```

Comment:
@hcwilhelm sorry I missed this one, let's please preserve the compatibility here, i.e. no need for two argument groups.

Reply:
@mijicd I think this is not possible because the compiler does not accept default parameters in a signature with a *-parameter. 

```
a parameter section with a `*`-parameter is not allowed to have default arguments
  final def xAdd[SK: Schema, I: Schema, K: Schema, V: Schema](
```


or do you have another suggestion on how to solve this?